src/main/java/dev/jbang/cli/Export.java

```diff
@@ -653,20 +654,23 @@ void renderBuildFile(BuildContext ctx, Path projectDir, String fullClassName) th
 		Template template = engine.getTemplate(templateRef);
 		if (template == null)
 			throw new ExitException(EXIT_INVALID_INPUT, "Could not locate template named: '" + templateRef + "'");
+		boolean isKotlin = ctx.getProject().getMainClass().endsWith("Kt");
 		String result = template
 								.data("group", group)
 								.data("artifact", artifact)
 								.data("version", version)
+								.data("kotlin", isKotlin ? "yes" : "no")
 								.data("description", prj.getDescription().orElse(""))
 								.data("repositories", repositories	.stream()
 																	.map(MavenRepo::getUrl)
 																	.filter(s -> !"".equals(s))
 																	.collect(Collectors.toList()))
 								.data("javaVersion", getJavaVersion(prj, false))
 								.data("gradledependencies", gradleify(depIds))
-								.data("fullClassName", fullClassName)
+								.data("fullClassName", fullClassName + (isKotlin ? "Kt" : ""))
```

Comment:
sholdn't this not already have Kt in its name at this stage?

Reply:
A Kotlin class called App is translated to Java byte as class AppKt.class.  So the addition of "Kt" is correct.